### DISCUSSION TITLE
Welcome to lobe-tts Discussions!

### DISCUSSION THREAD

**Discussion by canisminor1990** [lobehub/lobe-tts#38] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shakibdshy** [comment]
I have a few question, I actually explore your website but I didn't get to much data, So I ask you here:
Is it possible to do more natural conversation flow like a phone call where:
- I can speak and the AI responds
- I can interrupt the AI while it's speaking
- The AI can listen to me while it's speaking
- It's a continuous back-and-forth conversation

because of I am using STT and TTS togather. and I am using this hook: useSpeechRecognition and useSpeechSynthes
